website/core/Site.js

```diff
@@ -97,8 +97,8 @@ var Site = React.createClass({
         <head>
           <title>{title}</title>
           {
-            metaTags.map((tag) =>
-              <meta {...tag} />)
+            metaTags.map((tag, index) =>
+              <meta key={index} {...tag} />)
```

Comment:
This fixes [a warning](https://facebook.github.io/react/docs/multiple-components.html#dynamic-children).